modules/ROOT/pages/changelog.adoc

```diff
@@ -4,6 +4,78 @@
+* Show the calculated height and width of media embed elements in the `media` plugin dialog.
+* Removing an image that failed to upload from an empty paragraph would leave the paragraph without a padding br.
+* Allow a media embed element to be correctly resized when using the `media` plugin dialog by converting the media embed to a standalone iframe.
+* Inline alert in the "Search and Replace" dialog persisted when it wasn't necessary.
+* Context toolbars displayed the incorrect status for the `advlist` plugin buttons.
+* On iOS Safari, Korean characters were merging onto the previous line upon typing after inserting a newline by pressing Enter.
+* Initiating the editor with a table at the start would display resize handles even when the editor wasn't focused.
+* Backspace would in some situations delete the image after the cursor instead of before it as expected.
+* Directionality commands was setting the `dir` attribute on noneditable elements within a noneditable root.
+* The content of the dialog body could not be scrolled.
+* Some toolbar items were not rendering the `not-allowed` mouse cursor.
+* Formats were incorrectly applied to the closest editable element if the selection was in a noneditable context.
+* Formats were incorrectly removed to the closest editable element if the selection was in a noneditable context.
+* Formatter API `canApply` was not returning `false` when the selection was in a noneditable context.
+* When dragging image elements and dropping the image in the editor the `dragend` event would sometimes not fire on firefox.
+* It was possible to remove links in noneditable contents with the 'unlink' editor command.
+* Direction was not visually changing when using the Directionality plugin on an element which has the `direction` CSS property set.
+* Whitespace between transparent elements would incorrectly be converted into empty paragraphs.
+* Popups were not constrained within the scrollable container when in a shadow root.
+* Pressing arrow keys inside RTL elements would move the caret in an incorrect direction when moving over elements with the `contenteditable` attribute set to `false`.
+* Inserting table consecutively without focus in the editor would result in the table being inserted at the wrong position.
+* Pasting content into the editor would not fire `beforeinput` and `input` events.
+* In some cases, the exiting a `blockquote` element could fail when the cursor was positioned at the end of the `blockquote`.
+* Templates containing an `<html>` tag were not parsed before being rendered for preview.
+* Typing after deleting formatted content could remove a space at the start of the typing.
+* Invalid markup in Notification and Dialog close buttons.
+* In dialogs, an incorrect `aria-describedby` attribute caused the body of the dialog to be announced when using a screen reader.
+* The sticky toolbar would not be rendered correctly when transitioning from the custom editor view to the main view.
```

Comment:
```suggestion
* Text within anchor tags, <a>, presented with the Times New Roman font-family, ignoring the font family used in the original document.
* The sticky toolbar would not be rendered correctly when transitioning from the custom editor view to the main view.
```

Reply:
Missing changelog entry from TINY-9812

Reply:
This is a change to the PowerPaste Premium plugin and is set to be recorded as such in the PowerPaste changelog, as per the now-merged [PR176](https://github.com/tinymce/powerpaste2/pull/176/files).